content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Adding Linux worker nodes
+content_type: task
+weight: 50
+---
+
+<!-- overview -->
+
+This page explains how to add Linux worker nodes to a kubeadm cluster.
+
+## {{% heading "prerequisites" %}}
+
+* Obtain a [Windows Server 2019 license](https://www.microsoft.com/en-us/cloud-platform/windows-server-pricing) (or higher) in order to configure the Windows node that hosts Windows containers. If you are using VXLAN/Overlay networking you must have also have [KB4489899](https://support.microsoft.com/help/4489899) installed.
+* `curl.exe` is installed and present in the `PATH` environment variable.
+* A running kubeadm cluster created by `kubeadm init` and following the steps
+in the document [Creating a cluster with kubeadm](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/).
+
```

Comment:
You'll also need a server running Windows, and administrative access to the server (just having the license is not enough to proceed).

Reply:
attempted to reword the steps.